pynicotine/cli/application.py

```diff
@@ -101,6 +101,10 @@ def on_critical_error(self, _exc_type, exc_value, _exc_traceback):
     def on_critical_error_threading(args):
         raise args.exc_value
 
+    def shares_unavailable(self, shares):
+        for virtual_name, folder_path in shares:
+            log.add("• %s (%s)" % (virtual_name, folder_path))
```

Comment:
`"• \"%s\" at %s % (virtual_name, folder_path)`

Suggest use " quotes instead of brackets for the virtual name, because the planned shares commands will require names to be surrounded in quotes and it's helpful to have these matching syntax.